tools/LogCollector/source/omslinux_agentlog.py

```diff
@@ -411,6 +411,13 @@ def copyCommonFiles(omsLinuxType):
     cmd='cp /var/opt/omi/log/omi* /tmp/omslogs'
     out=execCommand(cmd)
     writeLogCommand(cmd)
+    scxExists=os.path.exists('/var/opt/microsoft/scx/log/scx.log')
+    if(scxExists == True):
+       cmd='cp /var/opt/microsoft/scx/log/scx* /tmp/omslogs'
+       out=execCommand2(cmd)
+       writeLogCommand(cmd)
+    else:
+       writeLogCommand('Skipping collection of scx.log as it does not exist')
     cmd='cp /var/opt/microsoft/scx/log/scx* /tmp/omslogs'
```

Comment:
need to delete this + next 2 lines otherwise you will try copying a second time regardless of file existence